internal/helpers/sort.go

```diff
@@ -0,0 +1,16 @@
+package helpers
```

Comment:
What if we start with resourceutils ? for functions to help process k8s resources. I worry that the sort package is too specific (narrow)  - I don't see  adding any more functions to it. if this assumption turns out wrong, we can refactor.

Reply:
Can I propose that we do the opposite? Let's start narrow with the `sort` package since that's what it currently does, and then we can always broaden its purpose once we have a reason to add a new function.